app/gui/src/project-view/components/ComponentMenu.vue

```diff
@@ -36,12 +36,12 @@ function closeDropdown() {
   isDropdownOpened.value = false
 }
 
-function openDocs(url: string) {
-  window.open(url, '_blank')
-}
-
-function readableBinding(binding: keyof (typeof graphBindings)['bindings']) {
-  return graphBindings.bindings[binding].humanReadable
+type Binding =
+  | keyof (typeof graphBindings)['bindings']
+  | keyof (typeof nodeEditBindings)['bindings']
+function readableBinding(binding: Binding) {
+  const bindings = { ...graphBindings.bindings, ...nodeEditBindings.bindings }
```

Comment:
Many bindings have generic names like `edit` that could conflict if we compose them like this. It would be safer for the caller of `readableBinding` to specify which binding set the binding name belongs to; the function could default to `graphBindings` if unspecified.

Reply:
Good point. Enforcing the correct type of optional argument is extremely annoying, so I just made it required. Not a big deal.